packages/mongo/connector.ts

```diff
@@ -71,7 +71,7 @@ export const createMongooseConnector = <TModel = any>(
     delete: async (selector) => {
       const deletedRawDocument = await MongooseModel.findOne(selector).exec();
       const deletedDocument = deletedRawDocument && deletedRawDocument.toJSON();
-      await MongooseModel.remove(selector);
+      await MongooseModel.remove(selector); // collection.remove is deprecated
```

Comment:
Ah yes I think I saw that.We might address this in the PR, I think the change is easy?